home_work_3.rb

```diff
@@ -0,0 +1,20 @@
+class Item
+  
+    attr_reader :whore, :prise, :weight, :breast_size
+    attr_writer :whore, :prise, :weight, :breast_size
+    
+    def initialize(whore, prise, weight, breast_size)
```

Comment:
```suggestion
    def initialize(name, prise, weight)
```
убрать breast_size и whore. Добавить name. Не мерджить это предложение, менять локально а потом пушить